### ISSUE TITLE
In glTF Tools 2.3.0, the three.js button is sticking out of the menu

### ISSUE THREAD

**Issue by cx20** [AnalyticalGraphicsInc/gltf-vscode#190]
I tried glTF Tools 2.3.0. However, the button of three.js seems to be out of the menu.

![image](https://user-images.githubusercontent.com/1833979/91181840-6191b780-e724-11ea-8277-dedd6cbdbfb3.png)

Unfortunately, I don't remember how it was in the previous version.

It may have something to do with the environment (Japanese OS, Japanese fonts, etc.). If you can tell me where to look, I'll confirm it.

**Comment by cx20** [comment]
I tried changing the `CSS` font part locally as a trial. `13px` seems to fit in the menu.
https://github.com/AnalyticalGraphicsInc/gltf-vscode/blob/7c4c080e69eae5441af4c3ab4018509fff7bea3f/pages/previewModel.css#L7

|font size|menuUI|
|:----|:----|
|font: 13px sans-serif;|![image](https://user-images.githubusercontent.com/1833979/91632132-bd588b00-ea19-11ea-9b0e-222ad2dccf3d.png)
|font: 14px sans-serif;|![image](https://user-images.githubusercontent.com/1833979/91632140-d5c8a580-ea19-11ea-8d77-4573cebd79ac.png)|
|font: 15px sans-serif;|![image](https://user-images.githubusercontent.com/1833979/91632144-db25f000-ea19-11ea-9beb-7b07087e758c.png)|

**Comment by cx20** [comment]
When we used `sans-serif`, the actual font is different between `Japanese fonts` and `English fonts`, so the size seems to be different.
https://twitter.com/clockmaker/status/859263366769946624

|lang|sample|
|:---:|:--------:|
|"ja"|![image](https://user-images.githubusercontent.com/1833979/91633112-a701fd80-ea20-11ea-8c79-51729ee4e3db.png)|
|"en"|![image](https://user-images.githubusercontent.com/1833979/91633114-a9fcee00-ea20-11ea-9ee4-6b4e9c69a2d0.png)|

**Comment by cx20** [comment]
Below is the result of trying `sans-serif` font using Chrome browser on _Japanese_ Windows 10.

https://w3g.jp/sample/css/font-family#sans-serif
![image](https://user-images.githubusercontent.com/1833979/91633481-7a9bb080-ea23-11ea-8f95-933e5dfe347c.png)

https://w3g.jp/sample/css/font-family#japanese
![image](https://user-images.githubusercontent.com/1833979/91633489-82f3eb80-ea23-11ea-89a6-f265a96babfa.png)

It seems that the `sans-serif` font is actually assigned to the `Meiryo` font.
I haven't tried it because I don't have an _English_ Windows 10, but I think the font assigned to `sans-serif` is probably different.

**Comment by emackey** [comment]
Thanks @cx20.  The sans-serif font in English is "Arial".

I think the box needs to be wider.  In particular, if [#mainUI.width](https://github.com/AnalyticalGraphicsInc/gltf-vscode/blob/7c4c080e69eae5441af4c3ab4018509fff7bea3f/pages/previewModel.css#L23) was `325px`, maybe it would fit?

Also I wonder if the UI is too wide now with 4 buttons across, maybe on thinner windows it could collapse to a 2x2 arrangement of buttons.

**Comment by cx20** [comment]
Thanks @emackey.

> he sans-serif font in English is "Arial".

I learned a lot. Thank you for the information.

> if #mainUI.width was 325px, maybe it would fit?

Yes, `325px` is just barely enough, so `330px` looked like a good choice. I've tried several combinations of main UI sizes and fonts.

![image](https://user-images.githubusercontent.com/1833979/91649593-a3b65280-eab0-11ea-9263-b4b0b6ab3aaa.png)
(If I make a table in markdown, the images in the cells are automatically resized, so I combined them into one image in Excel for comparison.)

> Also I wonder if the UI is too wide now with 4 buttons across, maybe on thinner windows it could collapse to a 2x2 arrangement of buttons.

I agree. we might want to change the layout to 2x2 if we have a smaller window size. if we have more glTF Viewers, we might need to think about 3x2, etc.
